pipeline/outputs/s3.md

```diff
@@ -28,6 +28,7 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
 | upload_timeout                | Whenever this amount of time has elapsed, Fluent Bit will complete an upload and create a new file in S3. For example, set this value to 60m and you will get a new file every hour.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | 10m                                     |
 | store_dir                     | Directory to locally buffer data before sending. When multipart uploads are used, data will only be buffered until the `upload_chunk_size` is reached.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | /tmp/fluent-bit/s3                      |
+| store_dir_limit_size                        | The size of the limitation for disk usage in S3. Limit the amount of s3 buffers in the `store_dir` to limit disk usage. Note: if you want to upload data to S3, using `store_dir_limit_size` instead of `storage.total_limit_size` which is used for other plugins.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | 0, which means unlimited                              |
```

Comment:
Sorry I think we can say this more succinctly and also explain it with:

```
S3 has its own buffering system, so use `store_dir_limit_size` instead of `storage.total_limit_size`.
```